### PR TITLE
Separate launch files

### PR DESCRIPTION
**Pull request by tjsos**

### CODE REVIEW

File: alpha_rise_bringup/launch/bringup_control.launch
Comment (mzhouURI) [inline alpha_rise_bringup/launch/bringup_control.launch:1]: I will call this bringup_mvp.launch, and we should launch mvp_control first.

File: alpha_rise_bringup/launch/bringup_sensors.launch
Comment (mzhouURI) [inline alpha_rise_bringup/launch/bringup_sensors.launch:1]: can you move dvl to the last? so when I am using this file for bench top test, I can quickly comment dvl out.

**Review by mzhouURI — CHANGES_REQUESTED** [review]
some minor edits are needed

**Comment by tjsos** [comment]
- Changed `bringup_control` to `bringup_mvp` & placed mvp_control first for launch.
- Moved DVL launch statement to the end of `bringup_sensors`

**Pull request merged**